README.md

```diff
@@ -42,10 +42,12 @@ You will also need to install:
       * You also need to install the `Command Line Tools` via Xcode. You can find this under the menu `Xcode -> Preferences -> Downloads`
       * This step will install `gcc` and the related toolchain containing `make`
   * On Windows:
-    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator).
+    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator). It will install:
+      * Python 2.7
+      * [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159)
```

Comment:
This is wrong. `windows-build-tools` installs the Microsoft **Visual C++** Build Tools 2015, not Microsoft Build Tools 2015. It also does some more configuration.

I think this paragraph should be left as it was. There's a link to the `windows-build-tools` page that has this explained on the top of the README, right after the screenshot.